water-effect/assets.js

```diff
@@ -0,0 +1,23 @@
+const baseUrl = import.meta.url.replace(/(\/)[^\/\\]*$/, '$1');
+
+export const glbUrlSpecs = {
+  particleGLBPath: {
+    ripple: `${baseUrl}../water-effect/assets/ripple.glb`,
```

Comment:
all of the assets should be added to this repo :
https://github.com/webaverse/procgen-assets
just like the terrain assets